backend/fms_core/resources/sample.py

```diff
@@ -278,6 +287,11 @@ def after_save_instance(self, instance, using_transactions, dry_run):
         super().after_save_instance(instance, using_transactions, dry_run)
         reversion.set_comment("Imported samples from template.")
 
+    def save_m2m(self, obj, data, using_transactions, dry_run):
+        sampleByProject = SampleByProject.objects.create(project=self.project, sample=obj)
```

Comment:
I would encourage you to use another attribute than self.project (self.project_obj maybe ? or change self.project to self.project_name to free up self.project) to store the reference to the project for this ... You don't need to assign it if you do not need sampleByProject.